org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/editor/WSearchEditor.java

```diff
@@ -261,7 +262,7 @@ public void setValue(Object value)
 	{
 		Object curValue = this.value;
         this.value = value;
-		if (value != null && !"".equals(String.valueOf(value)))
+		if (value != null && !"".equals(String.valueOf(value)) && lookup != null)
```

Comment:
I think both changes here is not needed for WSearchEditor. this editor is not going to work without a lookup.